src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -0,0 +1,165 @@
+        return new_position
+
+    def move_by(self, increment):
+        """
+        Move focuser by a given amount.
+
+        Args:
+            increment (int): distance to move the focuser, in encoder units.
+
+        Returns:
+            int: distance moved, in encoder units.
+        """
+        self._is_moving = True
+        try:
+            ini_pos = self.position
+            new_pos = int(ini_pos) + increment
+            self._send_command(f'M{int(new_pos):d}#', response_length=0)
+        finally:
+            # Focuser move commands block until the move is finished, so if the command has
+            # returned then the focuser is no longer moving.
+            self._is_moving = False
+
+        self.logger.debug(f"Moved by {increment} encoder units. Current position is {new_pos}")
+        return new_pos
+
+    ##################################################################################################
+    # Private Methods
+    ##################################################################################################
+
+    def _send_command(self, command, response_length=None):
+        """
+        Sends a command to the Focuser adaptor and retrieves the response.
+
+        Args:
+            command (string): command string to send (without newline), e.g. "P#"
+            response length (integer, optional, default=None): number of lines of response expected.
+                For most commands this should be 0 or 1. If None readlines() will be called to
+                capture all responses. As this will block until the timeout expires it should only
+                be used if the number of lines expected is not known (e.g. 'ds' command).
+
+        Returns:
+            list: possibly empty list containing the '\r' terminated lines of the response from the
+                adaptor.
+        """
+        if not self.is_connected:
+            self.logger.critical(f"Attempt to send command to {self} when not connected!")
+            return
+
+        # Clear the input buffer in case there's anything left over in there.
+        self._serial_port.reset_input_buffer()
+
+        # Send command
+        self._serial_io.write(command + '\r')
+
+        # Depending on which command was sent there may or may not be any further
+        # response.
+        response = self._serial_io.readline()
+
+        return response
+
+    def _initialise(self):
+        self._is_moving = True
+        try:
+            # Initialise the aperture motor. This also has the side effect of fully opening the iris.
+            self._initialise_aperture()
+            self.logger.info(f'{self} initialised')
+        finally:
+            self._is_moving = False
+
+    def _initialise_aperture(self):
+        self.logger.debug('Initialising aperture motor')
+        self._send_command('A00#')
+        self.logger.debug('Aperture initialised at maximum value')
```

Comment:
Is this more correct?

```suggestion
        self.logger.debug('Aperture initialised')
```